### ISSUE TITLE
Regarding context getting changed

### ISSUE THREAD

**Issue by chintanshrinath** [Vamsi995/Paraphrase-Generator#12]
When I am paraphrase with specific sentence that time the context of sentence getting changed. Is there any way to get control on that

**Code**
import numpy as np
def set_seed(seed):
  np.random.seed(seed)

set_seed(0)
sentence = "Accounting for physicians who are potential prescribers, the overall Outreach declined by 89%."

text =  "paraphrase: " + sentence + " </s>"

encoding = tokenizer.encode_plus(text, return_tensors="pt")
input_ids, attention_masks = encoding["input_ids"], encoding["attention_mask"]

outputs = model.generate(
    input_ids=input_ids, attention_mask=attention_masks,
    max_length=100,
    do_sample=True,
    top_k=50,
    top_p=0.95,
    #num_beams = 1,
    early_stopping=True,
    num_return_sequences = 5
)

for output in outputs:
    line = tokenizer.decode(output, skip_special_tokens=True,clean_up_tokenization_spaces=True)
    print(line)

**Input**
Accounting for physicians who are potential prescribers, the overall Outreach declined by 89%.

**Output**
Accounting for physicians who are potential prescribers, the overall outreach decreased 89%.
**Including physicians who are potential prescribers, the overall outreach grew by 89%.**
Recalling for physicians who are potential prescribing physicians, the overall outreach declined by 89%.
The overall outreach declined by 89% for doctors who are potential prescribers.
Accounting for physicians who are potential prescribers, the overall outreach decreased by 89%.

**Comment by Vamsi995** [comment]
This is a very interesting problem raised. Im not so sure on how to get a control on this but some work can be done in this direction.

**Comment by Vamsi995** [comment]
It mostly has to do with the training data.

**Comment by chintanshrinath** [comment]
Thanks Vamsi
